lib/seattleflu/id3c/cli/command/etl/redcap_det_childcare.py

```diff
@@ -0,0 +1,536 @@
+"""
+Process DETs for the Childcare REDCap projects.
+"""
+from dateutil.relativedelta import relativedelta
+from enum import Enum
+
+from .redcap import *
+from id3c.cli.command.etl import redcap_det
+from id3c.cli.redcap import is_complete, Record as REDCapRecord
+import logging
+from seattleflu.id3c.cli.command import age_ceiling
+
+
+LOG = logging.getLogger(__name__)
+
+
+class ChildcareProject():
+    id: int
+    lang: str
+    api_token_env_var: str
+    command_name: str
+
+
+    def __init__(self, project_id: int, lang: str, api_token_env_var: str, command_name: str) -> None:
+        self.id = project_id
+        self.lang = lang
+        self.api_token_env_var = api_token_env_var
+        self.command_name = command_name
+
+PROJECTS = [
+        ChildcareProject(23740, 'en', 'REDCAP_API_TOKEN', 'childcare')
+    ]
+
+LANGUAGE_CODE = {
```

Comment:
With the latest fixup, `LANGUAGE_CODE` is no longer used in this ETL. I don't _think_ we need this anymore?

Reply:
You know, I almost got rid of it but thought it is kind of nice to have a language associated with a REDCap project.
But correct; it is not being used now.

Reply:
Right, I'm going through the same thought process 😆 
Not sure where we would store the REDCap project language...